onnxruntime/test/testdata/squeezenet/model_opset11.with_runtime_opt.readme.txt

```diff
@@ -0,0 +1,3 @@
+model_opset11.with_runtime_opt.ort was generated by running the following command:
```

Comment:
It would be really nice to see the model structure in some form.

Reply:
this is derived from an existing test model. they can both be viewed with a tool like Netron.